app/models/configuration_script.rb

```diff
@@ -1,11 +1,4 @@
-class ConfigurationScript < ApplicationRecord
-  serialize :variables
-  serialize :survey_spec
-
-  acts_as_miq_taggable
-
-  belongs_to :inventory_root_group, :class_name => "EmsFolder"
-  belongs_to :manager,              :class_name => "ExtManagementSystem"
-
-  include ProviderObjectMixin
+class ConfigurationScript < ConfigurationScriptBase
+  # As a job template, I have a playbook
```

Comment:
playbook and job template terminology doesn't make sense at this level...Thinking the comment can just be removed.